src/promptflow-evals/promptflow/evals/evaluate/_batch_run_client/proxy_client.py

```diff
@@ -30,11 +31,13 @@ def run(self, flow, data, column_mapping=None, **kwargs):
         return ProxyRun(run=eval_future)
 
     def get_details(self, proxy_run, all_results=False):
-        run = proxy_run.run.result(timeout=BATCH_RUN_TIMEOUT)
+        batch_run_timeout = get_int_env_var(PF_BATCH_TIMEOUT_SEC, PF_BATCH_TIMEOUT_DEFAULT)
```

Comment:
Unlike CodeClient, ProxyClient relies on pfclient.run. The underlying Batch Engine honors the PF_BATCH_TIMEOUT_SEC environment variable and handles the timeout already. Therefore, it's not necessary to pass the timeout here. Once the Batch Engine times out, proxy_run.run.result() will return immediately.

Instead of making changes in this class, I suggest the following modifications in batch_run_context.py:

In __enter__, if PF_BATCH_TIMEOUT_SEC is not set, assign it a default value.
In __exit__, revert the PF_BATCH_TIMEOUT_SEC change.

Reply:
The original was already taking in the default batch timeout param - my only change here to try getting it from environment variables first.

Are you saying we should remove the timeout entirely in this method and move all of the timeout logic to enter/exit?

Reply:
Yes, that's correct!

Reply:
done